packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-checkbox/gux-form-field-checkbox.scss

```diff
@@ -68,6 +68,10 @@
   flex-direction: row;
   gap: var(--gse-ui-checkbox-gap);
 
+  .gux-label-beside {
+    align-self: center;
```

Comment:
Inspecting the elements, it looks like the checkbox and not the label is what needs to be centered. Here is a quick picture without this change:

![image](https://github.com/MyPureCloud/genesys-spark/assets/30388389/5bc9e02a-023b-4561-b566-811a41d46594)

Aligning the text to center of the not-centered checkbox is actually making the component almost a pixel larger (of whitespace).

Reply:
There is still some really funky stuff going on here. The heights of the items in the flexbox are not correct. In theory, if you can get them to be the correct hight then using align-self or align-items should work. I play around for about a half hour in the browser and got it really close. I think there is potentially some underlying height issues with the input.

The closest I got was by specifically setting the height on `.gux-input` to be equal to the input height and then using align-self to center. Still looked a bit off but got the overall component height to 18px. Without that height setting, it kept putting in a couple of extra pixels which tells me there is something weird going on... somewhere.

Reply:
@thomas-c-dillon  will create issue to re-review this.